src/backend/distributed/commands/grant.c

```diff
@@ -91,9 +91,14 @@ PreprocessGrantStmt(Node *node, const char *queryString,
 
 			if (priv->cols != NIL)
 			{
-				ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-								errmsg("grant/revoke on column list is currently "
-									   "unsupported")));
+				if (!DisablePreconditions)
```

Comment:
if this affects non-Citus tables it seems like a bug

Reply:
Yes, makse sense, we should be careful for such cases.

For example, we can check if `CollectGrantTableIdList` returns any tables that are in Citus' metadata such that `RelationIdListContainsCitusTableType(tableIdList, ANY_CITUS_TABLE_TYPE)` or such

Reply:
I removed the GUC, and it does not cause a failure. I guess I left that behind while I am testing something at start.

Reply:
Added more GUCs for different tasks related to vanilla testing. And renamed the old GUC.